mods/ra/rules/ai.yaml

```diff
@@ -434,6 +437,7 @@ Player:
 		ProtectionTypes: harv, mcv, mslo, gap, spen, syrd, iron, pdox, tsla, agun, dome, pbox, hbox, gun, ftur, sam, atek, weap, fact, proc, silo, hpad, afld, afld.ukraine, powr, apwr, stek, barr, kenn, tent, fix, fpwr, tenf, syrf, spef, weaf, domf, fixf, fapw, atef, pdof, mslf, facf
 		IgnoredEnemyTargetTypes: AirborneActor
 	UnitBuilderBotModule@naval:
+		IdleBaseUnitsMaximum: 20
```

Comment:
It's a bit weird to only do this for red alert, shouldn't this be increased for all mods? Perhaps the default increased as well? There are AI's who's squad sizes are larger than 12

Reply:
It was just for test purpose at that time. If you are sure this solve 
> It doesn't build even when it has no units

Then I am going to add it to all mod.

Reply:
the RA AI seemed to be functioning as intended. Although the exact numbers chosen aren't what I would expect for each AI

Reply:
I think I should add a comment on the "IdleBaseUnitsMaximum".